cli/src/publish.ts

```diff
@@ -57,6 +57,10 @@ export interface PublishOptions extends RegistryOptions {
 	 * Should use `yarn` instead of `npm`. Only valid with `packagePath`.
 	 */
     yarn?: boolean;
+    /**
+     * Enables publishing of web extensions. Only valid with `packagePath`.
```

Comment:
Consider to remove the "Only valid with" part (see comment above).

Reply:
Done